src/tests/clearCart.test.js

```diff
@@ -0,0 +1,64 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import cookieParser from 'cookie-parser';
+import app from '../index.js';
+import { cartServices } from '../services';
+
+chai.should();
+chai.use(chaiHttp);
+chai.use(cookieParser);
+const { expect } = chai;
+
+describe('Get /cart items ', function () {
+  const testUserLogin = {
+    email: 'testing@example.com',
+    password: 'Qwert@12345',
+  };
+
+  it('should  return 401, when not logged in', async function () {
+    const response = await chai.request(app).delete(`/cart/clear`);
+    expect(response).to.have.status(401);
+  });
+  it('should throw 404 error, when acesses wrong path', async function () {
+    const res = await chai
+      .request(app)
+      .post('/users/login')
+      .send(testUserLogin);
+    const { token } = res.body;
+    const response = await chai
+      .request(app)
+      .delete(`/ca-rt`)
+      .set({ Authorization: `Bearer ${token}` });
+    expect(response).to.have.status(404);
+  });
+
+  it('should return 500 error when there is a server error', async function () {
+    const res = await chai
+      .request(app)
+      .post('/users/login')
+      .send(testUserLogin);
+    const { token } = res.body;
+    const { clearCart } = cartServices;
+    cartServices.clearCart = () => {
+      throw new Error(errorMessage);
```

Comment:
Here you are mocking the clearCart function to throw an error. However, you have not defined the errorMessage variable. In my opinion, You should define it before using it.
You can place `  const errorMessage = 'Error message';`  right below the it()

Reply:
good idea, thanks